beacon-chain/rpc/eth/beacon/handlers.go

```diff
@@ -723,7 +723,27 @@ func (s *Server) PublishBlockV2(w http.ResponseWriter, r *http.Request) {
 		s.publishBlock(ctx, w, r, true)
 	}
 }
+func (s *Server) versionHeaderFromCurrentSlot() string {
+	// attempt to get the current fork information
+	ce := slots.ToEpoch(s.GenesisTimeFetcher.CurrentSlot())
```

Comment:
what do you guys think about this?

Reply:
If you want current fork information, then a slightly "cleaner" way is to use `CurrentFork()` from the blockchain service. There is one problem when doing it either way, but it's an edge case during the fork and clients should use v2 anyway. So I don't worry about it.

Reply:
I won't change it then 😅